site/content/en/docs/Installation/Creating Cluster/gke.md

```diff
@@ -79,6 +79,10 @@ gcloud container clusters create [CLUSTER_NAME] --cluster-version={{% k8s-versio
   --machine-type=n1-standard-4
 ```
 
+{{< alert title="Note" color="info">}}
```

Comment:
Please surround this with 

{{% feature publishVersion="1.14.0" %}}
...
{{% /feature %}}

so that it goes live with the next release (when we actually have an agones sdkserver that will run on windows nodes).